### DISCUSSION TITLE
questions for ogb.lsc

### DISCUSSION THREAD

**Discussion by zoneming** [snap-stanford/ogb#330] [PCQM4M-LSC]
problems:
    ImportError: cannot import name 'PCQM4MEvaluator' from 'ogb.lsc' (/home/anaconda3/envs/qgcn/lib/python3.9/site- 
   packages/ogb/lsc/__init__.py);
    
    ImportError: cannot import name 'PCQM4Mv2Dataset' from 'ogb.lsc' (/home/anaconda3/envs/qgcn/lib/python3.9/site- 
   packages/ogb/lsc/__init__.py);

dependencies:
    pytorch: 1.11.0
    pyg: 2.0.4
    dgl: 0.8.1
    ogb : 1.3.3
    pytorch-cluster     1.6.0
    pytorch-mutex          1.0 
    pytorch-scatter           2.0.9
    pytorch-sparse            0.6.13  
    pytorch-spline-conv       1.2.1

**Comment by weihua916** [comment]
Hi! You will need to install `rdkit`. See [here](https://pypi.org/project/rdkit-pypi/).

**Reply by zoneming** [reply]
thanks for your reply, its helpful. it is a problem of rdkit version indeed.
